apps/backend/src/datasources/postgres/FapDataSource.ts

```diff
@@ -381,6 +383,22 @@ export default class PostgresFapDataSource implements FapDataSource {
       });
   }
 
+  async getFapReviewsByCallAndFap(
+    callId: number,
+    fapId: number
+  ): Promise<Review[]> {
+    const fapReviews: ReviewRecord[] = await database
+      .select(['sr.*'])
+      .from('fap_reviews as sr')
```

Comment:
Why are the short names `sr` and `sp`? They don't seem to line up with the table names.

Reply:
I don't know where the 's' comes from. I decided it best to follow the naming convention established in the other queries.

Reply:
Thanks for clearing up the confusion. I'll pop in an issue to change that, don't think it's relevant to this PR.